scripts/darwin.sh

```diff
@@ -62,5 +66,23 @@ for disk in $DISKS; do
     echo "system: False"
   fi
 
+  partitions="`diskutil list $disk | get_device_identifiers`"
+  partitionmountpoints=""
+
+  for partition in $partitions; do 
+    partitioninfo="`diskutil info $partition`"
```

Comment:
In order to avoid calling `diskutil info` every time, which is an expensive operation, I suggest saving the output of running `mount`, and re-using it in every loop to find the mount points:

```
mount_output="`mount`"
mountpoint=`echo "$mount_output" | grep "^/dev/disk2s1" | cut -d ' ' -f 3`
```